bloom-housing/bloom
# refactor: add translations

# Pull Request Template

## Issue Overview

This PR addresses #3278


## Description

This PR moves translations out of UIC and into shared-helpers. You can see the UIC PR [here](https://github.com/bloom-housing/ui-components/pull/10).

## How Can This Be Tested/Reviewed?

Check out the deploy previews and ensure translations work! You can also check out the [UIC PR](https://github.com/bloom-housing/ui-components/pull/10) and link it locally to this PR.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
